yarn.lock

```diff
@@ -5979,7 +5979,7 @@ babel-plugin-macros@2.8.0, babel-plugin-macros@^2.0.0, babel-plugin-macros@^2.8.
     cosmiconfig "^6.0.0"
     resolve "^1.12.0"
 
-babel-plugin-remove-graphql-queries@2.7.2, babel-plugin-remove-graphql-queries@^2.7.2:
+babel-plugin-remove-graphql-queries@^2.7.2:
```

Comment:
These `yarn.lock` changes look a bit suss to me. Can you revert `yarn.lock` and then re-run `yarn` to see if these changes are actually needed?